File: java/8.5/example8.java
import java.util.Scanner;

public class example8 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the mark for test 1");
        Integer mark1 = Integer.parseInt(scanner.nextLine());
        System.out.println("Enter the mark for test 2");
        Integer mark2 = Integer.parseInt(scanner.nextLine());
        if (mark1 >= 90 && mark2 >= 90) {
            System.out.println("Well done you got top marks on both tests");
        } else if (mark1 >= 90 || mark2 >= 90) {
            System.out.println("Well odne you got top marks on one test");
        } else {
            System.out.println("You didn't quite get top marks on the tests, try again next time");
        }
    }
}

File: rust/src/8.5/example8.rs
fn main() {
    let mut mark1s = String::new();
    let mut mark2s = String::new();
    println!("Enter the first mark");
    std::io::stdin().read_line(&mut mark1s).unwrap();
    println!("Enter the seccond mark");
    std::io::stdin().read_line(&mut mark2s).unwrap();
    let mark1 = mark1s.trim().parse::<i32>().unwrap();
    let mark2 = mark2s.trim().parse::<i32>().unwrap();
    if mark1 >= 90 && mark2 >= 90 {
        println!("Well done you got top marks on both of your tests");
    } else if mark1 >= 90 || mark2 >= 90 {
        println!("You scored top marks on one of your tests");
    } else {
        println!("You didn't quite get top markes on the tests, try again next time");
    }
}